Source/DynamicTradeInterface/UserInterface/Columns/ColumnButtons.cs

```diff
@@ -47,6 +57,11 @@ public static void Draw(ref Rect rect, Tradeable row, Transactor transactor, ref
 			if (_settings?.GhostButtons == true && currentAmountToTransfer == 0 && Mouse.IsOver(rect.ExpandedBy(rect.width / 2, rect.height * 2)) == false)
 				return;
 
+			if (Event.current.rawType == EventType.MouseUp || Input.GetMouseButtonUp(0)) {
+				s_painting = false;
+				s_paintingDirection = PaintingDirection.None;
+			}
+
```

Comment:
Ideally this block would only run once per frame, but it's difficult to do that with how you've architected the mod.

Reply:
I will be looking into it closer soon, and testing the performance implications as well.
However this entire method is only ever called once per frame?

Reply:
This is called once per frame, per row, which is redundant because we're dragging across rows. We only need to call it once for all rows.